File: Examples/Example-ObjC/Example-ObjC/ViewController.m
//
//  ViewController.m
//  Example-ObjC
//
//  Created by Łukasz Jerciński on 15/03/2017.
//  Copyright © 2017 LiveChat Inc. All rights reserved.
//

#import "ViewController.h"
@import chatio;
@import MapKit;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.title = @"chat.io";
}

- (IBAction)openChat:(id)sender {
    //Presenting chat:
    [Chatio presentChatWithAnimated:YES completion:nil];
}

@end

File: Examples/Example-Swift/Example-Swift/ViewController.swift
//
//  ViewController.swift
//  Example-Swift
//
//  Created by Łukasz Jerciński on 06/03/2017.
//  Copyright © 2017 LiveChat Inc. All rights reserved.
//

import UIKit
import chatio
import MapKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        title = "chat.io"
    }
    
    @IBAction func openChat(_ sender: Any) {
        //Presenting chat:
        Chatio.presentChat()
    }
}

